[Merged by Bors] - Add aditional description to access assets tab (#496)
bevyengine/bevy-website: README.md

A local server should start and you should be able to access a local version of the website from there.

> ## Assets Page

Reviewer: ```suggestion
### Assets Page
```
I think this should be a sub section of the Zola build instructions

Author: It's just a way to jump line, but I put in the format of the upper topic.

Author: Done 👍

> To access Assets Page with successfully, it's necessary to go into the repository in **.\generate-assets** and run the command in terminal: <br>

Reviewer: ```suggestion
To build the Assets page, it's necessary to go into the **.\generate-assets** folder and run the command: <br>
```

I think this reads a bit better

Author: Done 👍

``generate_assets.sh``<br>

> If you are on Windows, maybe this cannot work, so It`s suggest recommending to use WSL or Git Bash.<br>

Reviewer: ```suggestion
If you are on Windows, It`s recommended to use WSL or Git Bash.<br>
```

Author: Done 👍

A local server should start and you should be able to access a local version of the website from there.

> ### Assets Page
> 
> 1. To build the Assets page, it's necessary to go into the **.\generate-assets** folder and run the command:
>    1. `generate_assets.sh`
> 2. If you are on Windows, it`s recommended to use WSL or Git Bash.
> 
> ## Assets generation
> 
> Assets are generated using data from /generate/assets and crates.io using their [datadump](https://crates.io/data-access) trough [cratesio-dbdump-lookup](https://github.com/alyti/cratesio-dbdump-lookup).

Reviewer 2: ```suggestion
### Assets, Errors, and Examples pages

These pages need to be generated in a separate step by running the shell scripts in the `generate-assets`, `generate-errors`, and `generate-wasm-examples` directories. On Windows, you can use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) or [git bash](https://gitforwindows.org/).

Assets are generated from a [crates.io data dump](https://crates.io/data-access) through [cratesio-dbdump-lookup](https://github.com/alyti/cratesio-dbdump-lookup).

Please note that when unpacked, the crates.io dump fills about 500 mb.
```

I would recommend something like this.

I think it's a bit odd that this is presented as a numbered list. The content doesn't really fit that presentation. There's really only one step.

The text below is pretty confusing too. It seems to be trying to accomplish the same thing as this PR to me, so I think we can roll that into the same section.

This also isn't specific to the assets page. The same pattern applies for errors and examples, so we might as well tell users about those too.

@IceSentry any thoughts, especially wrt the `## Assets generation` section?

Reviewer 2: I wouldn't be opposed to just dropping those sentences.

A local server should start and you should be able to access a local version of the website from there.

### Assets, Errors, and Examples pages

These pages need to be generated in a separate step by running the shell scripts in the `generate-assets`, `generate-errors`, and `generate-wasm-examples` directories. On Windows, you can use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) or [git bash](https://gitforwindows.org/).

Assets are generated from a [crates.io data dump](https://crates.io/data-access) through [cratesio-dbdump-lookup](https://github.com/alyti/cratesio-dbdump-lookup).

Please note that when unpacked, the crates.io dump fills about 500 mb.

> Please notice when unpacked, the crates.io dump fills about 500 mb.

Author: We agree with this suggestion @rparrett.
